### DISCUSSION TITLE
Not loading @components?

### DISCUSSION THREAD

**Discussion by zFlickan** [tailwindlabs/tailwindcss#12091] [Help] [answered]
Hi, 

I've just started to use Tailwind and React Remix v 2.0.
I went through the installationprocess on your page here: https://tailwindcss.com/docs/guides/remix

Since I've never used Tailwind before I don't know your classes. But I am fairly certain that the button that I'm trying to use first was blue and rounded using "btn btn-blue" (this was last week). But then I did something and the style dissapeared. And now when I install a new project the button, still, has only "stripping" style . Even in the freshly installed project, the button has just plain black text with no background or border at all.

I can use other classes, like w-full, uppercase and p-3.5. 
It feels like @base and @utilities are available, but not @components. But I haven't found any documentation to confirm that - I don't know which classes come from which part of Tailwind.

Or is the button-out-of-the-box really not supposed to look like anything?

Best regards
Frida

**Comment by wongjn** [comment]
`btn` and `btn-blue` would not be classes that Tailwind would generate CSS rules for with default configuration. The only class in the `@components` layer that Tailwind would generate a CSS rule for with default configuration would be [`container`](https://tailwindcss.com/docs/container).

**Reply by zFlickan** [reply]
Thank you for such quick reply!

So the button is not supposed to have any style - just black text?

**Reply by wongjn** [reply, marked as answer]
`<button>` elements would not have any style, just black text, with default configuration.

**Reply by zFlickan** [reply]
Ok, thank you!
